src/components/Cards/card.tsx

```diff
@@ -51,18 +39,62 @@ export default function Card({ props }: { props: CardProps }) {
           <h2 className="text-l font-bold">
             {props.title} <StatusPill status={props.status} />
           </h2>
-          <div>
-            <PrepopulatedRepairAttemptForm
-              props={props.repairRequestProps}
-              onSubmit={onSubmit}
-            ></PrepopulatedRepairAttemptForm>
+          <div className="text-left mb-2">
+            <h3 className="text-xl font-bold text-left">Details:</h3>
+            <ImageCarousel images={props.repairRequestProps.images} />
+            <div className="flex flex-row mt-2">
+              <div className="border-r-2 pr-2 mr-2">
+                {" "}
+                <h4 className="font-bold">Brand</h4>
+                <p className="mb-3">
+                  {props.repairRequestProps.itemBrand
+                    ? props.repairRequestProps.itemBrand
+                    : "-"}
+                </p>
+                <h4 className="font-bold">Material</h4>
+                <p className="mb-3">
+                  {props.repairRequestProps.itemMaterial
+                    ? props.repairRequestProps.itemMaterial
+                    : "-"}
+                </p>
+                <h4 className="font-bold">Hours Worked</h4>
+                {props.repairRequestProps.hoursWorked
+                  ? props.repairRequestProps.hoursWorked
+                  : "-"}
+              </div>
+              <div className="w-8/12">
+                <h4 className="font-bold">Description</h4>
+                <p className="mb-3">
+                  {props.repairRequestProps.description
+                    ? props.repairRequestProps.description
+                    : "-"}
+                </p>
+                <h4 className="font-bold">Repair Comment</h4>
+                <p className="mb-3">
+                  {props.repairRequestProps.repairComment
+                    ? props.repairRequestProps.repairComment
+                    : "-"}
+                </p>
+                <h4 className="font-bold">Parts Needed</h4>
+                <p className="mb-3">
+                  {props.repairRequestProps.spareParts
+                    ? props.repairRequestProps.spareParts
+                    : "-"}
+                </p>
+              </div>
+            </div>
+            <div className="my-2">
+              <h4 className="font-bold">Assigned to</h4>
+              {`${props.repairRequestProps.assignedTo?.firstName} ${props.repairRequestProps.assignedTo?.lastName}` ??
```

Comment:
If `assignedTo` was null, the null coalescing operator still wouldn't activate as "nullnull" is non-null